entrypoint.sh.release

```diff
@@ -77,6 +77,9 @@ EOF
 
 chmod +x /healthcheck.sh
 
+info=$(zerotier-cli info)
+echo "zerotier-cli info: $info"
```

Comment:
you can just inline this with the double quoted string. No need to store it in a var.

Reply:
Fixed. And nitpick away: it's no problem.